client/src/components/GetDetails/index.js

```diff
@@ -114,6 +114,7 @@ class GetDetails extends Component {
       const price = this.state.selected_price.concat(this.state.selected_currency);
       const inputs = {
         type: this.state.selected_itemType,
+        typeId: this.state.selected_itemType.id,
```

Comment:
/add-item endpoint only receive "type" in request so u have to set this.state.selected_itemType.id to type, remove typeId

Reply:
Okay I modified it in both cases. Take a look now.